src/GenFu/project.json

```diff
@@ -1,33 +1,30 @@
 ﻿{
-  "version": "1.1.0",
-  "description": "GenFu is a library you can use to generate realistic test data. It is composed of several property fillers that can populate commonly named properties through reflection using an internal database of values or randomly created data. You can override any of the fillers and give GenFu hints on how to fill properties.",
-  "authors": [ 
-        "misterjames",
-        "dpaquette",
-        "stimms"],
-  "tags": [ "" ],
-  "projectUrl": "",
-  "licenseUrl": "",
-    "frameworks": {
-        "net45": { },
-        "dnx451": {
-            "dependencies": {
-                "System.Linq": "4.0.1-beta-23516"
-            }
-        },
-        "dnxcore50": {
-            "dependencies": {
-                "System.Reflection": "4.1.0-beta-23516",
-                "System.Reflection.TypeExtensions": "4.0.1-beta-23409",
-                "System.Linq.Expressions": "4.0.11-beta-23516",
-                "System.Text.RegularExpressions": "4.0.11-beta-23516",
-                "Microsoft.CSharp": "4.0.1-beta-23516",
-                "System.IO.FileSystem": "4.0.1-beta-23516",
-                "System.Linq": "4.0.1-beta-23516"
-            }
-        }
-    },
-    "resource": [
-        "Resources/*.*"
-    ]
-}
+	"version": "1.1.2",
+	"description": "GenFu is a library you can use to generate realistic test data. It is composed of several property fillers that can populate commonly named properties through reflection using an internal database of values or randomly created data. You can override any of the fillers and give GenFu hints on how to fill properties.",
+	"authors": [
+		"misterjames",
+		"dpaquette",
+		"stimms"
+	],
+	"packOptions": {
+		"projectUrl": "http://genfu.io/",
+		"licenseUrl": "https://github.com/MisterJames/GenFu/blob/master/LICENSE.txt",
+		"repository": {
+			"type": "git",
+			"url": "git://github.com/MisterJames/GenFu"
+		},
+		"tags": [ "" ]
+	},
+	"buildOptions": {
+		"embed": "Resources/*.*"
+	},
+
+	"dependencies": {
+			"NETStandard.Library": "1.6.0"
+	},
+
+	"frameworks": {
+		"netstandard1.6": {},
```

Comment:
I got it down to netstandard1.3  lower than that and the System.Reflection.TypeExtensions doesn't support it.